settings.yaml

```diff
@@ -0,0 +1,35 @@
+repository:
+  # See https://developer.github.com/v3/repos/#edit for all available settings.
+
+  # The name of the repository. Changing this will rename the repository
+  name: sig-contributor-strategy
+
+  # A short description of the repository that will show up on GitHub
+  description: CNCF Special Interest Group on Contributor Strategy -- maintainer relations, building up contributors, governance, graduation, and more.
+
+  # A URL with more information about the repository
+  homepage: https://cncf.io/projects
+
+  # Collaborators: give specific users access to this repository.
+  # see /governance/roles.md for details on write access policy
```

Comment:
Also, we don't have this file yet. Should we remove this note until we do?

Reply:
I'd assumed we were going to add it.

Reply:
@justaugustus I'd keep this line commented for now, later we can just uncomment it to enable :)